### DISCUSSION TITLE
Git Lfs Bandwidth didnt reset

### DISCUSSION THREAD

**Discussion by Juanda9992** [community/community#38948] [Actions]
My GitHub lfs bandwidth exceed the free 1GB, my account says i have to wait  5 days to reset my quota. I have been waited 6 days and nothing has been reset, but now it says i have to wait 30 days to reset my quota.

![image](https://user-images.githubusercontent.com/64047011/201492738-f601d68e-f526-413d-b86a-ad06f7318a7d.png)

**Comment by jfullstackdev** [comment]
maybe Support can help 

https://support.github.com/contact/account

**Comment by hcoona** [comment]
I also meet this issue.

**Comment by h0lybyte** [comment]
Oh no, I am still at 0 days, I hope I dont run into this issue :/

**Comment by totallynotdavid** [comment]
Same issue! Has anyone been able to find a solution?

**Comment by Ben-Drucker** [comment]
I did the following and it fixed the issue: 

- Sent a message to [GitHub Support](https://support.github.com/contact/account) explaining that my LFS bandwidth was not being reset.
- I received a reply about half-a-day later explaining that they manually triggered a job to reset my bandwidth usage.
- Sure enough, when I rechecked my bandwidth usage, it was reset to zero.
- Unfortunately, the solution was not manually doable, but at least GitHub support acted relatively quickly.

**Reply by JaneOrih** [reply]
the link [Github Support](https://support.github.com/contact/account) is not working. my bandwidth is also not resetting. my Github link https://github.com/JaneOrih . I exceeded my git lfs bandwidth
